jujugui/static/gui/src/app/assets/javascripts/react-click-outside.js

```diff
@@ -21,10 +21,23 @@ function enhanceWithClickOutside(WrappedComponent) {
     componentDidMount: function componentDidMount() {
       this.__wrappedComponent = this.refs.wrappedComponent;
       document.addEventListener('click', this.handleClickOutside, true);
+      // XXX ant 9 Feb 2016 Add Esc keybind to also close the current component
```

Comment:
That said it does perform the same as the click method by calling `this.refs.wrappedComponent.handleClickOutside(e);` which is a method on the component.